weblate/templates/list-translations.html

```diff
@@ -11,6 +11,7 @@
 <th class="col-fuzzy">{% trans "Review" context "Number of strings needing review" %}</th>
 <th class="col-checks">{% trans "Checks" context "Number of failing checks" %}</th>
 <th class="col-suggestions">{% trans "Suggestions" context "Number of suggestions" %}</th>
+<th class="col-comments">{% trans "Comments" contect "Number of comments" %}</th>
```

Comment:
Typo, should be context here. That's why the tests fail

Reply:
Having the test set up this wai is pure gold.

Thank you very much for pointing that out!